Scripts/Interfaces/Entity/IMainMenu.cs

```diff
@@ -39,6 +39,30 @@ public interface IMainMenu : IMenu, IUsesMenuActions, ISelectTool
 		/// Does this menu have focus?
 		/// </summary>
 		bool focus { get; }
+
+		/// <summary>
+		/// Add a settings menu to this menu
+		/// </summary>
+		/// <param name="provider">The object providing the settings menu</param>
+		void AddSettingsMenu(ISettingsMenuProvider provider);
+
+		/// <summary>
+		/// Remove a settings menu from this menu
+		/// </summary>
+		/// <param name="provider">The object which provided the settings menu</param>
+		void RemoveSettingsMenu(ISettingsMenuProvider provider);
+
+		/// <summary>
+		/// Add a settings menu to this menu item
```

Comment:
Add a setting menu to this menu item, or add a menu item to this settings menu?

Reply:
It's adding a menu item to the item menu menu item item, of course!